.github/updatecli.d/bump-golang.yml

```diff
@@ -39,75 +39,87 @@ sources:
     spec:
       command: echo {{ requiredEnv "GO_MINOR" }}
 
-  latestGoVersion:
-    name: Get Latest Go Release
+  latestMicrosoftGoVersion:
+    name: Get Latest Microsoft Go Release
     kind: githubrelease
     dependson:
       - minor
     transformers:
-      - trimprefix: go
+      - trimprefix: v
     spec:
-      owner: golang
+      owner: microsoft
       repository: go
       token: '{{ requiredEnv "GITHUB_TOKEN" }}'
       username: '{{ requiredEnv "GITHUB_ACTOR" }}'
       versionfilter:
         kind: regex
-        pattern: go1\.{{ source "minor" }}(\.(\d*))?$
+        pattern: v1\.{{ source "minor" }}\.(\d*)(-\d*)$
+
+  golangVersion:
+    name: Get golang version
+    dependson:
+      - latestMicrosoftGoVersion
```

Comment:
Could we please add a comment here explaining why we look at the `microsoft/go` repo for the Golang version instead of the upstream Go repo?  That way this makes sense to someone looking at this configuration outside the context of this PR.  Thanks!

Reply:
Sure, let me do that.

However, I want to be specific here. Does `git blame` not answer those questions? I prefer using `git log` and `blame` to understand what changes were made, why they were made, and to gain more context.

But maybe it's just me